addons/web_tour/models/tour.py

```diff
@@ -1,30 +1,108 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, fields, models
+from odoo import api, fields, models, Command
+import json
+import base64
 
 
 class Tour(models.Model):
-
     _name = "web_tour.tour"
     _description = "Tours"
-    _log_access = False
+    _order = "sequence, name, id"
+
+    name = fields.Char(string="Tour Name", required=True)
+    step_ids = fields.One2many("web_tour.tour.step", "tour_id")
+    url = fields.Char(string="Starting Url")
```

Comment:
```suggestion
    url = fields.Char(string="Starting Url", default="/odoo")
```
Since most tours begin on the same page, we could maybe use it as default to simplify tours creation everywhere? :thinking: 

Also, does it make sense for a tour to have no starting url?
If it doesn't, maybe the field should be required too.

Reply:
Some of them are compute in the JS (mainly website's ones). So I cannot make it required in the DB because this tours will get their url at the start of the tour service.
But the default value is a good idea 😃